app/gateway-oss/2.4.x/hybrid-mode.md

```diff
@@ -290,6 +290,27 @@ cluster_cert_key = data-plane.crt
 `cluster_server_name` specifies the SNI (Server Name Indication extension) to use for data plane
 connections to the control plane through TLS. When not set, data plane will use `kong_clustering` as the SNI.
 
+## Revocation check of Data Plane certificates
+
+When Kong is running Hybrid mode with PKI support, the Control Plane can be configured to
+optionally check for revocation status of the conecting Data Plane certificate.
+
+The supported method is through Online Certificate Status Protocol (OCSP) respoders.
+In addition, Data Plane certificates issued must contain the "Certificate Authority Information Access" extension
+which points to the URI of which the OCSP responder may be reached from the Control Plane.
```

Comment:
I feel many CAs won't let you touch raw individual extension so user may not aware about what `Certificate Authority Information Access` is. Maybe let's just say:
```suggestion
In addition, Data Plane certificates issued must contain the URI of OCSP responder may be reached from the Control Plane.
```
How do you think?

Reply:
The suggested sentence is not grammatically correct. Suggest:

In addition, Data Plane certificates issued must contain the URI of OCSP responder that may be reached from the Control Plane.

Reply:
@fffonion @tlblessing thanks for the review.

My feeling is that in case the user manages their own internal CA, at least we can give some hint on what they should be looking for. So I will be proposing to combine those two:

> In addition, Data Plane certificates issued must contain the "Certificate Authority Information Access" extension that contains the URI of OCSP responder that may be reached from the Control Plane.

What do you folks think?

Reply:
@dndx I think that's fine but I'd remove the double quotes.

Also, @javierguerragiraldez  is working on this same source file (different section) in PR https://github.com/Kong/docs.konghq.com/pull/2770#pullrequestreview-633713301 fyi.

Reply:
Made your edit in commit 50f5747. There are 4 instances of "In addition", so I removed that as well. Even more refinements could be made here.

Reply:
Yes this looks good to me.